### PR TITLE
237 resultbrowser refactor image loading

### PR DESCRIPTION
**Pull request by jeanNimb**
[Issue](https://github.com/Xceptance/XLT/issues/237)

### CODE REVIEW

File: resultbrowser/src/css/default.css [hunk 1]
```diff
@@ -50,11 +51,11 @@ body {
...
-    background-image: url(data-url:../images/progress.gif);
-    width: 24px;
-    height: 24px;
-    float: right;
-    margin: 2px;
+    margin: 10px;
+    position: fixed;
+    top: 0;
+    right: 0;
+    z-index: 2147483647;
```
Comment (h-arlt) [inline resultbrowser/src/css/default.css:58]: As we have full control over our CSS, there is no need to use such a high value for z-index. IMO, a value of 10 should be sufficient.

File: resultbrowser/src/css/default.css [hunk 2]
```diff
@@ -81,7 +82,7 @@ body {
     background-color: white;
 }
 
-#menu > ul > li {
+#menu>ul>li {
```
Comment (h-arlt) [inline resultbrowser/src/css/default.css:85]: Please revert this change and all the others that removed the spaces from the child combinator. It's really hard to read selectors formatted like this.

File: resultbrowser/src/css/default.css [hunk 3]
```diff
@@ -595,3 +606,30 @@ table.key-value-table td.value .csep {
...
+  /* Safari */
+  @-webkit-keyframes spin {
+    0% { -webkit-transform: rotate(0deg); }
+    100% { -webkit-transform: rotate(360deg); }
+  }
+  
+  @keyframes spin {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+  }
```
Comment (h-arlt) [inline resultbrowser/src/css/default.css:635]: Please do not remove the final newline character at end-of-file.

**Review by h-arlt — CHANGES_REQUESTED** [review]
See comments inline.

File: resultbrowser/src/css/default.css (hunk 3 above)
Comment (jeanNimb) [reply resultbrowser/src/css/default.css]: Done

File: resultbrowser/src/css/default.css (hunk 2 above)
Comment (jeanNimb) [reply resultbrowser/src/css/default.css]: Done

File: resultbrowser/src/css/default.css (hunk 1 above)
Comment (jeanNimb) [reply resultbrowser/src/css/default.css]: Done

**Review by h-arlt — APPROVED** [review]
